bspump/elasticsearch/connection.py

```diff
@@ -26,12 +25,18 @@ def __init__(self, connection, index, max_size):
 		self.FailLogMaxSize = connection.FailLogMaxSize
 		self.FilterPath = connection.FilterPath
 
-	def consume(self, _id, data):
-		self.Items.append((_id, data))
-		self.Capacity -= len(data)
+	def consume(self, data):
```

Comment:
`data` are actually not a data but a data feeder (which is generator).
Can you please rename it that way since it is a bit confusing to read it without it.